cms/djangoapps/contentstore/tests/test_course_settings.py

```diff
@@ -942,6 +943,16 @@ def test_allow_unsupported_xblocks(self):
         XBlockStudioConfigurationFlag(enabled=True).save()
         self.assertIn('allow_unsupported_xblocks', CourseMetadata.fetch(self.fullcourse))
 
+    def test_allow_adaptive_learning_configuration(self):
+        """
+        Tests to make sure that depending on how the flag is set, whether or not
+        the the adaptive_learning_configuration is located in the filtered list.
+        """
+        AdaptiveLearningEnabledFlag(enabled=False).save()
```

Comment:
@macornwell What happens when you remove this line? We have to make sure that the test still passes in that case (as the flag is [disabled](https://github.com/edx/django-config-models/blob/master/config_models/models.py#L112) by default). Let's shoot for just doing

```python
        self.assertNotIn('adaptive_learning_configuration', CourseMetadata.fetch(self.fullcourse))
        AdaptiveLearningEnabledFlag(enabled=True).save()
        self.assertIn('adaptive_learning_configuration', CourseMetadata.fetch(self.fullcourse))
```

in the body of this test.